tests/modeling/test_modeling.py

```diff
@@ -1937,11 +1937,6 @@ def test_zo_abs_on_nl_elim(testdata):
             'ETAB2 = (EXP(ETA(2))**THETA(5) - 1)/THETA(5)',
             'CL = TVCL*EXP(ETAB1)\nV = TVV*EXP(ETAB2)',
         ),
-        (
```

Comment:
Couldn't this stay with 'ETA(1)' instead of 'eta(1)' ?

Reply:
True, perhaps a too small difference. You don't have to change it.

Reply:
Ok, thank you!